tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -473,27 +448,27 @@ else if (args[i].toLowerCase().equals("-help")) {
     for ( ; i < args.length; i++) {
       // continue the loop where the last one left off
       // Print documentation line on System.out
-      ToolIO.out.println("\n****** SANY2 " + version + "\n") ;
+      ToolIO.out.println("\n****** SANY2 " + SANY_BUILD_VERSION + "\n") ;
 
       // Get next file name from command line; then parse,
       // semantically analyze, and level check the spec started in
       // file Filename leaving the result (normally) in Specification
       // spec.
       SpecObj spec = new SpecObj(args[i], null);
       // check if file exists
-      if (FileUtil.createNamedInputStream(args[i], spec.getResolver()) != null) 
```

Comment:
The createNamedInputStream api returns an inputStream which is supposed to be closed (Closable)